### PR TITLE
Add ViewCommand implementation details to DG

### LINKED ISSUE #182: DG: Add ViewCommand implementation details

### PR DESCRIPTION
**Pull request by RezwanArefin01**
Resolves #182. 
In the process also makes some consistency changes in `GradeCommand`.

### CODE REVIEW

File: docs/diagrams/GradeCommandSequenceDiagram.puml [hunk 1]
```diff
@@ -20,14 +21,19 @@ activate GradeCommand
...
-Model --> GradeCommand: moduleClass
+Model --> GradeCommand: focusedClass
 deactivate Model
 
 GradeCommand -> Model: getFilteredStudentList()
 activate Model
 Model --> GradeCommand: lastShownList
 deactivate Model
 
+GradeCommand -> Util: parseStudentFromIndex
```
Comment (Berted) [inline docs/diagrams/GradeCommandSequenceDiagram.puml:32]: Would it be better to show `lastShownList` as a parameter to improve clarity?

File: docs/diagrams/GradeCommandSequenceDiagram.puml [hunk 2]
```diff
@@ -20,14 +21,19 @@ activate GradeCommand
...
+activate Util
+Util --> GradeCommand: oldStudent
+deactivate Util
+
 GradeCommand -> StudentC: getUpdatedStudent
```
Comment (Berted) [inline docs/diagrams/GradeCommandSequenceDiagram.puml:37]: Similarly, would it be better to show `oldStudent` parameter?

**Review by Berted — CHANGES_REQUESTED** [review]
Looks OK. However, I find the diagram can perhaps have a bit more clarity in what parameters are being passed?

File: docs/diagrams/GradeCommandSequenceDiagram.puml (hunk 2 above)
Comment (RezwanArefin01) [reply docs/diagrams/GradeCommandSequenceDiagram.puml:37]: Only oldStudent parameter? I didnt show it as there are other parameters.

File: docs/diagrams/GradeCommandSequenceDiagram.puml (hunk 1 above)
Comment (RezwanArefin01) [reply docs/diagrams/GradeCommandSequenceDiagram.puml:32]: Then we'd need to include the index too, which will make it more cluttered.

Comment (Berted) [reply docs/diagrams/GradeCommandSequenceDiagram.puml:32]: Fair point, but I'd argue we'd need more clarity, but it should be sufficient for now. Maybe add this as an issue. Will resolve for now.

**Review by Berted — APPROVED** [review]
LGTM!

**Pull request merged**